react-app/src/components/Opensearch/main/Settings/Visibility.tsx

```diff
@@ -7,16 +7,25 @@ import * as UI from "@/components";
 type Props<T extends UI.OsTableColumn> = {
   list: T[];
   onItemClick: (field: string) => void;
+  hiddenColumns: T[];
 };
 
 export const VisibilityPopover = <T extends UI.OsTableColumn>(
-  props: Props<T>
+  props: Props<T>,
```

Comment:
Can we destructure props? I know this isn't your change, but since we're here, let's do it